cli/tests/pcluster/validators/test_instances_validators.py

```diff
@@ -33,22 +33,22 @@
         (
             "TestComputeResource",
             {
-                "t2.micro": InstanceTypeInfo(
-                    {"InstanceType": "t2.micro", "VCpuInfo": {"DefaultVCpus": 4, "DefaultCores": 2}}
+                "t4g.micro": InstanceTypeInfo(
```

Comment:
FYI: t4g is arm based instances. Currently, multi-architecture cluster is not supported. But since this is unit test, I am ok with the change